scripts/qemu-dm-wrapper

```diff
@@ -19,6 +19,8 @@ path = "/sbin:/usr/sbin:/bin:/usr/bin"
 
 xenstore_write_cmd = "xenstore-write"
 
+cgroup_slice = ''
```

Comment:
It doesn't. But if in future someone wants to change it to be something different, they only need to worry about changing that one line.